src/components/Documents/DocumentTableRow.jsx

```diff
@@ -30,7 +35,17 @@ const DocumentTableRow = ({ document }) => {
 
   const handleShowDocumentLocal = async (urlToOpen) => {
     const urlFileBlob = await getBlobFromSolid(session, urlToOpen);
-    window.open(urlFileBlob);
+    const newWindow = window.open(urlFileBlob);
```

Comment:
Are these changes neede for the profile page to work? If not, I think we can save memory optimizations on the documents table for another PR. There are better ways to handle this, and we dont need to hash them out here.

Reply:
No, profile can work without this. It's just that the memory used for the document page that's being opened wouldn't be free up. It'll just be a minor change really, I'll have this in another PR.